requirement-checker/src/RequirementCollection.php

```diff
@@ -18,9 +18,10 @@
 use Traversable;
 
 /**
- * The code in this file must be PHP 5.3+ compatible as is used to know if the application can be run.
- *
  * @private
+ * @see \Symfony\Requirements\RequirementCollection
+ * @package symfony/requirements-checker
+ * @license MIT (c) Fabien Potencier <fabien@symfony.com>
```

Comment:
from the license file
https://github.com/symfony/requirements-checker/blob/master/LICENSE

> The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.